staging.midrc.org/etlMapping.yaml

```diff
@@ -378,4 +380,6 @@ mappings:
             fn: set
           - name: loinc_system
             src: loinc_system
-            fn: set
\ No newline at end of file
+            fn: set
+    parent_props:
```

Comment:
Remove the "parent_props" here and add it to the injecting_props section for dataset above